packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -235,8 +235,12 @@ export class CodeFlowLogin {
                   error.message
                 )
             );
-            const accessToken = await this.login();
-            return accessToken;
+            await this.logout();
```

Comment:
Will this trigger `statusChanged`? Or, how to ensure the expiration triggers correct status changed callback.

Reply:
No, codeFlow workFlow does not know about notify logic. Notify logic is handled by loginProvider. So now statusChange is handled by code after ```codeFlowInstance.getToken()```.

Reply:
I see, make sense.